test/test_config.py

```diff
@@ -472,35 +461,38 @@ def test_bad_realm_conf(self):
             r"the host test_host_realm3 got an invalid realm \(Realm3\)!"
         )
         self.assert_any_cfg_log_match(
-            "Configuration in realm::Realm1 is incorrect; from: cfg/config/realm_bad_member.cfg:5"
+            r"hosts configuration is incorrect!"
         )
         self.assert_any_cfg_log_match(
-            r"\[realm::Realm1\] as realm, got unknown member 'UNKNOWNREALM'"
+            "Configuration in realm::Realm1 is incorrect; from: cfg/config/realm_bad_member.cfg:5"
         )
         self.assert_any_cfg_log_match(
-            "Error : More than one realm are set to the default realm"
+            r"\[realm::Realm1\] as realm, got unknown member 'UNKNOWNREALM'"
         )
         self.assert_any_cfg_log_match(
-            "Error: the realm configuration of yours hosts is not good because there is more "
-            r"than one realm in one pack \(host relations\):"
+            "realms configuration is incorrect!"
         )
         self.assert_any_cfg_log_match(
-            "the host test_host_realm2 is in the realm Realm2"
+            re.escape(
+                "Error: Hosts exist in the realm <Realm \"name\"=u'Realm2' /> "
```

Comment:
It is one of your modification that reintroduced the object rather than the name :wink: 
I will replace with the realm name...